### DISCUSSION TITLE
wh41 air quality sensor

### DISCUSSION THREAD

**Discussion by Mareado13** [Daveiano/weewx-wdc#272] [Q&A]
I have a wh41 air quality sensor
How do I get the data on the main page and on the graphs? It would be nice if I could see the current data, the 1 hour average and the 24 hour AQ average.
I'm a bit lost
https://meteodosrius.es/weewx/index.html

**Comment by Daveiano** [comment]
There is a dedicated wiki page on [Adding new observations](https://github.com/Daveiano/weewx-wdc/wiki/Adding-new-observations).

It should cover all your needs, please feel free to ask if anything is not answered there!
